src/dvclive/dvc.py

```diff
@@ -105,6 +104,24 @@ def make_dvcyaml(live):
         dvcyaml["params"] = [os.path.relpath(live.params_file, live.dir)]
     if live._metrics:
         dvcyaml["metrics"] = [os.path.relpath(live.metrics_file, live.dir)]
-    if live._metrics or live._plots or live._images:
-        dvcyaml["plots"] = [os.path.relpath(live.plots_dir, live.dir)]
+    plots = []
+    if live._metrics:
+        plots.append(
+            os.path.relpath(
+                os.path.join(live.plots_dir, Metric.subfolder), live.dir
```

Comment:
Not even sure (didn't try) if it's possible to have `dvclive/plots` and then `dvclive/plots/sklearn_plots/confusion_matrix`. 

As this gets autogenerated I didn't see any downside to including each subdirectory as a separate entry in `plots`.